src/lib/SideBar.svelte

```diff
@@ -21,15 +20,37 @@
 		{ icon: 'fab fa-discord', info: 'Discord', activity: null },
 		{ icon: 'fab fa-github', info: 'GitHub', activity: null },
 	];
-
-	let activeInfo = null;
+	let activeInfo = null; // Tracks currently visible info.
+	let hideTimeout = 0; // Timeout for hiding info panel.
 
 	function showInfo(info) {
+		clearTimeout(hideTimeout);
+		hideTimeout = 0;
 		activeInfo = info;
 	}
-
 	function hideInfo() {
-		activeInfo = null;
+		// Prevents multiple timers and hides the info panel after 400ms unless interrupted.
+		clearTimeout(hideTimeout);
```

Comment:
Can we simplify this out? I notice that there are 2 codepaths that call hideInfo, with one codepath being new in this commit and the other pre-existing. Is the pre-existing one still required? Maybe we can remove it altogether.